pdf/extract.go

```diff
@@ -68,6 +55,23 @@ func ExtractText(inputPath, password string, pages []int) (string, error) {
 	return text, nil
 }
 
+func extractPageText(page *unipdf.PdfPage) (string, error) {
+	// Get page streams.
+	streams, err := page.GetContentStreams()
+	if err != nil {
+		return "", err
+	}
+
+	var pageContent string
+	for _, stream := range streams {
+		pageContent += stream
+	}
+
+	// Extract page text.
+	parser := unicontent.NewContentStreamParser(pageContent)
+	return parser.ExtractText()
```

Comment:
Sure. I've changed the code to use the extractor package. I also updated the unidoc dependency so that unicli uses the latest version of the extractor.